### ISSUE TITLE
Train Binary Classification

### ISSUE THREAD

**Issue by iefengjian** [zhiweichen0012/CATR#3]
Hello, thank you very much for your work! My data set only has one category, which has positive samples and background samples. During training, I changed the loss function to "F.multilabel_soft_margin_loss", but the loss_csm never converged. Is there any way to solve it?

**Comment by zhiweichen0012** [comment]
Hello, thank you very much for your interest in our work! 

First, you need to make sure that the file structure of your dataset is correct; second, some parameters in the model (e.g., G, p and K, etc., as mentioned in the paper) need to be adjusted according to your own dataset; and lastly, if you change the loss function, please make sure that the modification is correct. 

I hope the above reply is helpful to you.

**Comment by iefengjian** [comment]
Thanks for your reply, I will try it.
